CHANGELOG.md

```diff
@@ -10,12 +12,16 @@
 
 ### Documentation 📝
 
+* Updated the docstring for the MS operation to include the new `theta` parameter, explaining its use and effects on the gate operation.
```

Comment:
The wording sounded like it was documentation added for an existing feature, but it is actually a new feature, so probably change it to:
```suggestion
* The MS operation now includes a new `theta` parameter to specify the entanglement ratio of the qubits.
```